sympy/polys/subresultants_qq_zz.py

```diff
@@ -53,6 +53,15 @@
 called the resultant of f, g and serves as a criterion of whether
 the two polynomials have common roots or not.
 
+In sympy the resultant is computed with the function
+resultant(f, g, x). This function does _not_ evaluate the
+determinant of sylvester(f, g, x, 1); instead, it returns
+the last member of the subresultant prs of f, g!
+
+Caveat: If Df = degree(f, x) and Dg = degree(g, x), then:
+
+resultant(f, g, x) = (-1)**(Df*Dg) * resultant(g, f, x).
```

Comment:
I am planning to change the way the sign of resultant is computed in dup_prs_resultant, so maybe the wording could be modified accordingly in anticipation.